client/client_test.go

```diff
@@ -33,18 +34,38 @@ func TestAddLeaf(t *testing.T) {
 	if err := mysql.CreateTree(logID, env.DB); err != nil {
 		t.Errorf("Failed to create log: %v", err)
 	}
-
 	client := New(logID, env.ClientConn)
 
 	if err := client.AddLeaf(ctx, []byte("foo")); err != nil {
 		t.Errorf("Failed to add Leaf: %v", err)
 	}
```

Comment:
Verify that the leaf was really added in the log? An empty AddLeaf() function would pass this test.